<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDialogSalarySvyatkoviDni</class>
 <widget class="QDialog" name="UDialogSalarySvyatkoviDni">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Довідник &quot;Всяткові дні&quot;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_pidrozdily">
     <property name="title">
      <string>Святкові дні</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_yearFilter">
          <property name="text">
           <string>Рік</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_yearFilter">
          <property name="minimum">
           <number>2000</number>
          </property>
          <property name="maximum">
           <number>2100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_svyatkoviDni"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_reload">
       <property name="text">
        <string>Поновити</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/reload-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/reload-256.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_create">
       <property name="text">
        <string>Створити</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/edit_add-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/edit_add-256.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>Видалити</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/edit_remove-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/edit_remove-256.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../uPictRc.qrc"/>
 </resources>
 <connections/>
</ui>
